underscore.js

```diff
@@ -435,6 +435,15 @@
     if (_.has(result, key)) result[key]++; else result[key] = 1;
   });
 
```

Comment:
For this PR I just made the change in `_.pairs`, but the helper function allows quick checks in other underscore functions as well, as @jdalton mentioned (i.e. `_.toArray`).